src/lib/utils/wagmi.ts

```diff
@@ -30,35 +30,37 @@ export const ledgerConnector = new IFrameEthereumConnector({
   options: {},
 })
 
+const projectId = process.env.NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID!
+
 const connectors = () =>
   connectorsForWallets([
     {
       groupName: 'Recommended',
       wallets: [
         safeWallet({ chains }),
-        metaMaskWallet({ chains }),
-        rainbowWallet({ chains }),
-        argentWallet({ chains }),
+        metaMaskWallet({ chains, projectId }),
+        rainbowWallet({ chains, projectId }),
+        argentWallet({ chains, projectId }),
         coinbaseWallet({
           appName: 'Index Coop',
           chains,
         }),
-        ledgerWallet({ chains }),
-        okxWallet({ chains }),
+        ledgerWallet({ chains, projectId }),
+        okxWallet({ chains, projectId }),
       ],
     },
     {
       groupName: 'Others',
       wallets: [
-        walletConnectWallet({ chains }),
+        walletConnectWallet({ chains, projectId }),
         braveWallet({ chains }),
-        trustWallet({ chains }),
+        trustWallet({ chains, projectId }),
       ],
     },
-  ])().concat([ledgerConnector])
+  ])() // .concat([ledgerConnector])
```

Comment:
We currently can't support the ledger connector lib - as it's on an old version of wagmi.

Created an issue to ask for support: https://github.com/LedgerHQ/ledger-live-wagmi-connector/issues/28

Reply:
That repo is deprecated so I guess they wont be upgrading to the new wagmi version.